build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Would `ubi-minimal` work for us?

Reply:
```
registry.access.redhat.com/ubi8/ubi-minimal            latest              db39bd4846dc        3 weeks ago         106MB
registry.access.redhat.com/ubi8/ubi                    latest              fd73e6738a95        3 weeks ago         231MB
```

Reply:
No, because we need the openssl tools for the ES auto-provisioning scripts.